test/tests/parallel/set3/mta/smtp_delivery.bats

```diff
@@ -154,15 +154,14 @@ function setup_file() {
 
   assert_output --partial '1 <added@localhost.localdomain>'
   assert_output --partial '6 <user1@localhost.localdomain>'
-  assert_output --partial '1 <user1@localhost.localdomain>, orig_to=<root>'
```

Comment:
By "testing `orig_to`", do you mean "testing whether aliases are working"?

Reply:
> do you mean "testing whether aliases are working"?

Probably :)

I have been a maintainer for years but don't actually use DMS myself.

This test was written before I refactored it. It may not have been intentional originally, but as the diff shows, there is some extra coverage there?

Reply:
> I have been a maintainer for years but don't actually use DMS myself.

Well you're a hero.

I'll look into whether this is testing aliases and whether they're being tested anywhere else. Then I'll add a new test for aliases if need be.